libnetdata/queue/queue.h

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+
+#ifndef NETDATA_QUEUE_H
+#define NETDATA_QUEUE_H 1
+
+#include "stdlib.h"
+#include "pthread.h"
```

Comment:
We don't need these includes.

Reply:
They are for unit tests otherwise compilation error ioccured.